### PR TITLE
Add missing argument for hmac timestamp validation

### PR DESCRIPTION
**Pull request by aaannz**

### CODE REVIEW

**Comment by coolo** [comment]
hmm, how can we cover this code path?

**Comment by aaannz** [comment]
Right. `t/api/03-auth.t` is the place. Let me check if I can do some header magic there.

**Comment by aaannz** [comment]
Tests added

**Comment by coolo** [comment]
that sums up to 0.2% - wow

File: t/api/03-auth.t [hunk 1]
```diff
@@ -69,6 +69,82 @@ my $ws = OpenQA::WebSockets->new;
 
 my $ret;
 
+subtest 'access limiting for non authenticated users' => sub() {
```
Comment (okurz) [inline t/api/03-auth.t]: why the '()' on the 'sub'?

In this case I guess helper sub's would be nicer to look at.

**Pull request merged**